.github/workflows/check-if-migration-is-required.yml

```
@@ -1,4 +1,4 @@
-name: check if migration is required
+name: check if migration or lockfile are missing
```

I think simpler is better here...

```suggestion
name: check migration
```